pkg/pillar/cmd/diag/diag.go

```diff
@@ -807,6 +813,21 @@ func printOutput(ctx *diagContext, caller string) {
 		}
 	}
 
+	// Print evaluation platform status if available
+	if ctx.evalStatus.IsEvaluationPlatform {
+		evalLevel := "INFO"
+		if !ctx.evalStatus.AllowOnboard {
+			evalLevel = "WARNING"
+		}
+		if ctx.evalStatus.RebootCountdown > 0 {
+			evalLevel = "WARNING"
+		}
+		ctx.ph.Print("%s: evaluation: platform=%t slot=%s phase=%s onboard=%t note=%s\n",
+			evalLevel, ctx.evalStatus.IsEvaluationPlatform,
```

Comment:
No need to print ctx.evalStatus.IsEvaluationPlatform since you check that in the if on L817